corehq/apps/hqwebapp/static/hqwebapp/js/multiselect_utils.js

```diff
@@ -24,9 +24,9 @@ hqDefine('hqwebapp/js/multiselect_utils', [
         });
     };
 
-    var _renderAction = function (buttonId, buttonClass, buttonIcon, text) {
+    var _renderAction = function (buttonId, buttonClass, buttonIcon, text, disabled=false) {
```

Comment:
I didn't know js had default params! Feeling like a 🦖

Reply:
Kind of annoying the spacing style is different than for python. [Apparently](https://eslint.org/docs/rules/space-infix-ops) there's an option to allow `foo|0` without a space, but no exception for equals signs in function params.

Reply:
Yeah definitely a bit annoying, but I'd expect nothing less. That's a bummer that eslint doesn't have an option for the equals sign spacing in parameters.